paddle/fluid/operators/elementwise/elementwise_add_op.cc

```diff
@@ -127,6 +127,16 @@ REGISTER_OP_CPU_KERNEL(
                               paddle::platform::complex<float>>,
     ops::ElementwiseAddKernel<paddle::platform::CPUDeviceContext,
                               paddle::platform::complex<double>>);
+REGISTER_OP_CPU_KERNEL(
+    elementwise_add_one_dnn,
+    ops::ElementwiseAddKernel<paddle::platform::CPUDeviceContext, float>,
+    ops::ElementwiseAddKernel<paddle::platform::CPUDeviceContext, double>,
+    ops::ElementwiseAddKernel<paddle::platform::CPUDeviceContext, int>,
```

Comment:
It's the registration of the reference kernel, there can be models where the original operator was replaced with the custom one but since it's type is double it will fallback to reference.